### PR TITLE
Fixed issue #17256: No warning about adjusted question codes is shown when importing a group which holds questions with already used question codes

### PR DESCRIPTION
**Pull request by gabrieljenik**

### CODE REVIEW

**Comment by Shnoulle** [comment]
If i don't make error : when import old survey : we have the complete list of update. It's great

File: application/views/questionGroupsAdministration/import_view.php [hunk 1]
```diff
@@ -37,6 +37,20 @@
...
+                    <!-- Warnings -->
+                    <?php if (count($aImportResults['importwarnings'])>0): ?>
+                        <h2 class="warning"><?php eT("Warnings");?>:</h2>
+                        <ul  class="list-unstyled">
+                            <?php
+                                foreach ($aImportResults['importwarnings'] as $warning)
+                                { ?>
+                                <li><?php echo $warning; ?></li>
+                                <?php
+                            } ?>
```
Comment (olleharstedt) [inline application/views/questionGroupsAdministration/import_view.php:50]: endforeach; instead? You're using endif; below already. :)

File: application/helpers/admin/import_helper.php [hunk 2]
```diff
@@ -587,6 +587,10 @@ function XMLImportGroup($sFullFilePath, $iNewSID, $bTranslateLinksFields)
     LimeExpressionManager::RevertUpgradeConditionsToRelevance($iNewSID);
     LimeExpressionManager::UpgradeConditionsToRelevance($iNewSID);
 
+    if (count($aQuestionCodeReplacements)) {
+        array_unshift($results['importwarnings'], "<span class='warningtitle'>" . gT('Attention: Several question codes were updated. Please check these carefully as the update  may not be perfect with customized expressions.') . '</span>');
```
Comment (olleharstedt) [inline application/helpers/admin/import_helper.php:591]: Feel free to split call into multiple lines.

```php
array_unshift(
    $results['importwarnings'],
    "<span class='warningtitle'>"
    . gT('Attention: Several question codes were updated. Please check these carefully as the update  may not be perfect with customized expressions.')
    . '</span>'
);
```

**Review by olleharstedt — CHANGES_REQUESTED** [review]

File: application/views/questionGroupsAdministration/import_view.php (hunk 1 above)
Comment (gabrieljenik) [reply application/views/questionGroupsAdministration/import_view.php]: Yes, we were just reusing the same style which was on the file :)

**Comment by Shnoulle** [comment]
Not possible to add the list of updated code ?

See with old lss : 

Capture d’écran_2021-05-03_15-23-05

**Pull request merged**